tlatools/org.lamport.tlatools/javacc/tla+.jj

```diff
@@ -3192,217 +3029,6 @@ AssertStep() : {
     }
 }
 
-// SyntaxTreeNode
-// NumerableStep() : {
```

Comment:
If we want SANY to do better proof parsing, are we going to need this production?

It's mentioned in a few other places:
 - SyntaxTreeConstants.java has an unused `N_NumerableStep` constant
 - Generator.java has an unused `generateNumerableStep` method

Is there any chance we'll miss those later?

Reply:
Unlikely, looking at the places in `TlaPlusParserOutputTranslator.java` that required more work to draw structure out of the parser output, I see these trouble areas:
1. The use-or-hide construct
2. The assume/prove construct
3. The TAKE proof step type
4. The PICK proof step type

None of which have much to do with what this rule appears to do.

Reply:
Ok, as far as I can tell, the functionality of this production has been superseded by the more general `Step()` production. We should make a point to clean up SyntaxTreeConstants.java and Generator.java in a separate PR.